code/game/turfs/turf.dm

```diff
@@ -180,11 +180,11 @@
 	if(L)
 		qdel(L)
 
-/turf/proc/TerraformTurf(path, defer_change = FALSE, keep_icon = TRUE, ignore_air = FALSE)
-	return ChangeTurf(path, defer_change, keep_icon, ignore_air)
+/turf/proc/TerraformTurf(path, defer_change = FALSE, ignore_air = FALSE)
+	return ChangeTurf(path, ignore_air)
```

Comment:
It doesn't look like these arguments line up correctly - `ignore_air` here lines up with `defer_change` of `ChangeTurf`